lte/gateway/c/session_manager/PipelinedClient.h

```diff
@@ -135,6 +135,11 @@ class PipelinedClient {
     const std::string &ip_addr,
     const std::vector<std::string> &static_rules,
     const std::vector<PolicyRule> &dynamic_rules) = 0;
+
+  virtual bool set_upf_session(
```

Comment:
Please provide a short documentation in the header file :)

Reply:
Sure, I'll do the same.